sources/packages/api/src/services/atbc/atbc.controller.spec.ts

```diff
@@ -0,0 +1,97 @@
+require("../../../env_setup");
+import { closeDB, setupDB } from "../../testHelpers";
+import { Connection } from "typeorm";
+import * as faker from "faker";
+import { Student, User } from "../../database/entities";
+import { StudentService, ArchiveDbService, ATBCService } from "../../services";
+import { KeycloakConfig } from "../../auth/keycloakConfig";
+import { KeycloakService } from "../auth/keycloak/keycloak.service";
+import { ConfigService } from "../config/config.service";
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import { Test, TestingModule } from "@nestjs/testing";
+import * as request from "supertest";
+import { ATBCCreateClientResponse } from "../../types";
+
+describe("Test student model", () => {
```

Comment:
Please consider renaming this to something for meaningful for ATBC context, like "Test ATBC Controller"